_posts/2022-06-23-PC-Engines-Firmware-v4.17.0.1.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: post
+title:  "PC Engines apu coreboot Open Source Firmware v4.17.0.1"
+date:   2022-05-27
+categories: Firmware
+---
+
+# PC Engines apu coreboot Open Source Firmware v4.17.0.1
+
+## Key changes
+
+Mainline:
+
+### Added
+
+- Support for APU7 (APU3 variant with 2.5GbE i225 NICs)
+
+### Changed
+
+- **Rebased** with official coreboot repository **commit** 5eda52a
+- **Updated sortbootorder** to **v4.6.24** adding support for APU7
+
+## Statistics
+
+![Files Changed](https://cloud.3mdeb.com/index.php/s/FIXME/preview)
```

Comment:
Missing chart

Reply:
@sulewskiprzemyslaw Could you fill these in as well?

Reply:
The link to the chart has been added

Reply:
Thanks!